ntoskrnl/ps/query.c

```diff
@@ -941,7 +943,11 @@ NtQueryInformationProcess(
             }
             _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
             {
-                /* Get the exception code */
+                if (NT_SUCCESS(Status))
```

Comment:
OK but since we're manipulating the handle it's usually better in this circumstance to do the check against it.
```suggestion
                if (DebugPort)
```